web/apps/dashboard/.env.example

```
@@ -6,7 +6,7 @@ CLERK_SECRET_KEY=
 NEXT_PUBLIC_CLERK_SIGN_IN_URL=/auth/sign-in
 NEXT_PUBLIC_CLERK_SIGN_UP_URL=/auth/sign-up
 
-# Planetscale
+# Database
 
 DATABASE_HOST="aws.connect.psdb.cloud"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)